registry.json

```diff
@@ -393,6 +393,36 @@
       "documentation": "https://www.icondev.io/docs/icon-json-rpc-v3"
     }
   },
+  {
+    "id": "aptos",
+    "name": "Aptos",
+    "displayName": "Aptos",
+    "coinId": 637,
+    "symbol": "APT",
+    "decimals": 8,
+    "chainId": "32",
+    "blockchain": "Aptos",
+    "derivation": [
+      {
+        "path": "m/44'/637'/0'/0'/0'"
+      }
+    ],
+    "curve": "ed25519",
+    "publicKeyType": "ed25519",
+    "explorer": {
+      "url": "https://explorer.aptoslabs.com",
+      "txPath": "/txn/",
+      "accountPath": "/account/",
+      "sampleTx": "91424546",
```

Comment:
This tx hash looks incorrect

Reply:
It's not a tx hash for aptos: https://explorer.aptoslabs.com/txn/91424546

Reply:
hash looks more friendly to me 👀